awesome_cart/public/js/client/awc.erpnext.adapter.js

```diff
@@ -1002,6 +1002,8 @@ $(function () {
 					});
 				}
 			}
+			// trigger FullStory to record custom data
+			$("body").trigger("awc-power-user-settings", data);
```

Comment:
When does this get triggered? Are we sure it's triggered always? To make sure that Fullstory is tracking everything?

Reply:
yeah this code is executed on all the pages